src/DeviceInput/InputDevices/deviceSourceManager.ts

```diff
@@ -171,7 +171,10 @@ export class DeviceSourceManager implements IDisposable {
      * @param deviceSlot "Slot" or index that device is referenced in
      */
     private _removeDevice(deviceType: DeviceType, deviceSlot: number) {
-        delete this._devices[deviceType][deviceSlot];
+        if (this._devices[deviceType]?.[deviceSlot]) {

```

Comment:
I'm not sure I get this change. How is it possible for this if check to be false?

Reply:
Answered offline